### ISSUE TITLE
can't send files with huawei p20 pro

### ISSUE THREAD

**Issue by hueldoeu** [redsolution/xabber-android#939]
with lg g5 everthing works but it has only android 8 (some essential apps don't work) and huawei p20 pro has android 10.

**Comment by andrewnenakhov** [comment]
Thy this somewhat improved version https://www.xabber.com/rel/client/android/last/xabber.apk

(we have actually abandoned the existing xabber for android because of many internal architectural problems and are rewriting it anew with the experience received while developing iOS and web versions. It might take a while, but it'll come)

**Comment by hueldoeu** [comment]
yup, works perfectly. ah man. now i have to change the settings again.

i really think that google is trying to suck the worlds titties and drink our milk. i mean there are so many restrictions. as a developer i am scared to program something for android because i don't have full access to the mobiles.

google really only wants google-software on "their" smartphones. i can't use k-9 mail for gmail starting june 2022. one mailprovider says that google wants the whole control over their servers really if they want a certificate. this is insane!

when android started 2008 i thought "wow! a fully linux-smartphone. now i can use linux on a small device too" but now i see that if you have stock rom android on it, you have so many restrictions, maybe one day there won't be smartphones with microSD port. maybe lineageOS will give us more opportunity.

**Comment by hueldoeu** [comment]
> Thy this somewhat improved version https://www.xabber.com/rel/client/android/last/xabber.apk
> 
> (we have actually abandoned the existing xabber for android because of many internal architectural problems and are rewriting it anew with the experience received while developing iOS and web versions. It might take a while, but it'll come)

when i change from light skin to dark skin in the settings, xabber won't open anymore and i have to reinstall it. i'll remain on light skin (non-dark)

**Comment by hueldoeu** [comment]
is there still some place where i can help for the german translation?

**Comment by hueldoeu** [comment]
when i send a message from miranda ng windows 10 to xabber dev, xabber crashes!

**Comment by hueldoeu** [comment]
i have installed xabber from play store called xabber beta. works great! byebye xabber dev. dear @andrewnenakhov 
andrewnenakhov are you still interested in bitcoins?
